patches/p-cancelable@4.0.1.patch

```diff
@@ -0,0 +1,23 @@
+diff --git a/index.js b/index.js
+index c48660ea0f74f3d61c2ab8d57f139f7c115e91de..aec70a44cea7077996c5126b0cdccb69f1996c66 100644
+--- a/index.js
++++ b/index.js
+@@ -50,7 +50,17 @@ export default class PCancelable {
+ 
+ 			const onCancel = handler => {
+ 				if (this.#state !== promiseState.pending) {
+-					throw new Error(`The \`onCancel\` handler was attached after the promise ${this.#state.description}.`);
++					console.error(
++						`The \`onCancel\` handler was attached after the promise ${this.#state.description}.`,
++					);
++
++					reject(
++						new Error(
++							`The \`onCancel\` handler was attached after the promise ${this.#state.description}.`,
++						),
++					);
```

Comment:
I don't think it makes sense to reject again if the Promise is already rejected? It cannot reject twice, right?

Reply:
Good point. So, this will lead to nothing happening, right? 🤔 
Should I just log it? This edge case happens when a request is resolved after several failures, meaning that the relevant `Promise` is already resolved.

Reply:
fixed